Executable/Program.cs

```diff
@@ -196,6 +173,55 @@ internal static void Main(string[] args)
             }
         }
 
+        internal static void GetInfo(out OS os, out string windowsDirectory, out string macDirectory)
```

Comment:
`os` is an `out` parameter for this method, but I'm not seeing it get assigned anywhere.

Reply:
I must've deleted this by accident. Addressed in commit 0dcc903